src/store/Network.ts

```diff
@@ -268,6 +274,12 @@ export default {
       }
     },
 
+    RENTAL_FEE({ rootGetters }): Promise<RentalFees> {
```

Comment:
- start all REST querying store actions with `REST_`
- recommendation to use name `REST_NETWORK_RENTAL_FEES`